tests/Directory.Build.props

```diff
@@ -7,6 +7,9 @@
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
     <WarningLevel>0</WarningLevel>
+    <CollectCoverage>true</CollectCoverage>
```

Comment:
is this good? Because this is generic to any OS

Reply:
this enables for all tests. I don't know what you mean by generic to any OS.